packages/app-builder/src/routes/ressources+/scenarios+/$scenarioId+/builder-options.tsx

```diff
@@ -35,5 +38,6 @@ export async function loader({ request, params }: ActionFunctionArgs) {
     dataModel,
     databaseAccessors: accessors.databaseAccessors,
     payloadAccessors: accessors.payloadAccessors,
+    hasAccess: hasAnyEntitlement(entitlements),
```

Comment:
`hasAccess` is too vague, can it be named more properly? For example: `hasValidLicense`, `hasPrivateOperatorAccess` (I have a preference for `hasValidLicense`)